Storage/transaction_policy.h

```diff
@@ -211,6 +211,7 @@ class Synchronous final {
 
   PERSISTER& persister_;
   MutationJournal& journal_;
+  TransactionMetaFields& transaction_meta_fields_;
```

Comment:
If I understand it right and this is the reference owned by the object outside transaction policy, perhaps call it such? Ex. `transaction_meta_fields_reference_`?